src/main/java/com/box/sdk/legalholds/LegalHoldPolicy.java

```diff
@@ -0,0 +1,529 @@
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import com.box.sdk.BoxAPIConnection;
+import com.box.sdk.BoxAPIException;
+import com.box.sdk.BoxAPIResponse;
+import com.box.sdk.BoxJSONRequest;
+import com.eclipsesource.json.JsonObject;
+import com.google.gson.Gson;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+
+/**
+ * Legal Hold Policy.
+ */
+public final class LegalHoldPolicy {
+
+    private static final String LEGAL_HOLD_POLICY_URL_PATH = "legal_hold_policies";
+
+    private LegalHoldPolicy() {
+    }
+
+    /**
+     * Constructor.
+     *
+     * @param api        The agent that communicates with API
+     * @param policyName The name of the policy
+     * @return Legal Hold Policy message
+     * @throws MalformedURLException If error in URL
+     */
+    public static LegalHoldPolicy.Info createLegalHoldPolicy(BoxAPIConnection api, String policyName) throws
+            MalformedURLException {
+
+        return createLegalHoldPolicy(api, policyName, "", "", "");
+    }
+
+    /**
+     * Constructor.
+     * @param api        The agent that communicates with API
+     * @param policyName The name of the policy
+     * @param policyDescription The description of the policy
+     * @return Legal Hold Policy message
+     * @throws MalformedURLException If error in URL
+     */
+    public static LegalHoldPolicy.Info createLegalHoldPolicy(BoxAPIConnection api,
+                                                             String policyName,
+                                                             String policyDescription) throws MalformedURLException
+    {
+        return createLegalHoldPolicy(api, policyName, policyDescription, "", "");
+    }
+
+
+    /**
+     * Create Legal Hold Policy.
+     *
+     * @param api             agent that makes calls against API
+     * @param policyName      name of the policy
+     * @param description     description of policy (limit to 500 characters)
+     * @param filterStartedAt start date
+     * @param filterEndedAt   end date
+     * @return Legal Hold Policy
+     * @throws MalformedURLException If error in URL
+     */
+    public static LegalHoldPolicy.Info createLegalHoldPolicy(BoxAPIConnection api,
+                                                             String policyName,
+                                                             String description,
+                                                             String filterStartedAt,
+                                                             String filterEndedAt)
+            throws MalformedURLException {
+
+        BoxJSONRequest request = new BoxJSONRequest(api,
+                new URL(api.getBaseURL() + LEGAL_HOLD_POLICY_URL_PATH),
+                "POST"
+        );
+
+        JsonObject jsonRes = (new JsonObject()).add("policy_name", policyName);
+
+        if (!description.equals("")) {
```

Comment:
In Java, it tends to be better to invert this check:

`!"".equals(description)`

The reason being that, if `description` is passed in as `null`, this will throw a `NullPointerException`. There are also `StringUtils` methods that can handle the equality check, doing a null check internally.